README.md

```diff
@@ -1,2 +1,95 @@
-# navigation
-Repository for the discussion and research in to navigating from page to page whilst staying in immersive mode.
+# UA initiated immersive sessions
+
+## Motivation
+
+Current workflow to [request a WebXR session](https://www.w3.org/TR/webxr/#dom-xr-requestsession) requires user gesture activation. 2D Pages usually provide a button that users click or tap to trigger immersive mode.
+
+Some scenarios like in-vr navigation require entering immersive mode automatically without user gesture activation. This necessitates a standard mechanism for UAs to grant immersive sessions.
+
+## Design Goals
+
+The goal of this repository is to discuss and design a simple API for UAs to grant immersive sessions without an explicit user activation in specific scenarios.
+
+## Non Goals
+
+Define how a Virtual Reality or Augmented Reality browser would work [in alignment with the WebXR spec](https://github.com/immersive-web/webxr/blob/master/explainer.md#non-goals)
+
+## User Scenarios
+
+1. Immersive navigation, allowing UAs to implement seamless transitions between interlinked VR and AR content. Use cases include:
```

Comment:
I think we should explicitly call out cases that do and don't align with existing web concepts like CORS.  In particular, same-origin navigation could be handled differently than cross-origin.  Most of the examples listed here could have different requirements and behaviors.